build.sh

```diff
@@ -81,16 +81,30 @@ linux_build() {(
 	arch="`uname -m`"
 	check radare2-${VERSION}-${arch}.deb && return
 	prepare radare2-${VERSION} tmp/osx-pkg
-	msg "Building Debain GNU/Linux package..."
+	msg "Building Debian GNU/Linux package..."
 	sys/debian.sh >> ${LOG}
 	output sys/debian/radare2/*.deb
 )}
 
 docker_linux_build() {(
 	arch="$1"
 	check radare2_${VERSION}_${arch}.deb && return
-	cd docker-linux-${arch} || exit 1
-	make
+	cd tmp/radare*
+#	${CWD}/dockcross --image dockcross/linux-${arch} ./configure --with-compiler=${arch} --host=${arch}
+	${CWD}/dockcross --image dockcross/linux-${arch} sys/build.sh
```

Comment:
Is there a cleaner way to trim those other than using ugly `${text// /}` hacks, sed or similar?